genealogy/geneweb/Portfile

```
@@ -1,109 +1,127 @@
+
+name                geneweb
+version             7.1-beta
+revision            0
+github.setup        ${name} ${name} ${version} v
+categories          genealogy
+maintainers         {pguyot @pguyot} openmaintainer
+license             GPL-2
+
+description         Sophisticated Genealogy System
+long_description    GeneWeb is sophisticated genealogy software with a \
+                    web interface.  It provides efficient relationship and consanguinity \
+                    computing, support for very large database (i.e., hundreds of thousands \
+                    of entries), and more.
+
+homepage            https://geneweb.tuxfamily.org/wiki/GeneWeb
+extract.suffix      .zip
+checksums           rmd160  3745f4476378fb90cd21f628ae6a2e93dd0dae21 \
+                    sha256  527bc5a59e378cad144449800d214c641d775f5269716f3c40e68c5de87eab9a \
+                    size    14465775
+
+depends_lib         port:ocaml \
+                    port:ocaml-dune \
+                    port:ocaml-calendars \
+                    port:camlp5 \
+                    port:ocaml-camlp-streams \
+                    port:ocaml-cppo \
+                    port:ocaml-fmt \
+                    port:ocaml-jingoo \
+                    port:ocaml-markup \
+                    port:ocaml-num \
+                    port:ocaml-ocamlformat \
+                    port:ocaml-ounit \
+                    port:ocaml-ppx_blob \
+                    port:ocaml-ppx_deriving \
+                    port:ocaml-ppx_import \
+                    port:ocaml-stdlib-shims \
+                    port:ocaml-syslog \
+                    port:ocaml-unidecode \
+                    port:ocaml-uri \
+                    port:ocaml-uucp \
+                    port:ocaml-uunf \
+                    port:ocaml-uutf \
+                    port:ocaml-zarith
+
+use_parallel_build  no
+
+set gwuser          ${name}
+set gwgroup         ${name}
+set logdir          ${prefix}/var/log
+set langdir         ${prefix}/share/${name}/gw
+set dbdir           ${prefix}/var/db/${name}
+
+add_users ${gwuser} group=${gwgroup} \
+                    home=${dbdir} \
+                    realname=Geneweb
+
+startupitem.create  yes
+startupitem.user    geneweb
+startupitem.executable \
+                    ${prefix}/share/${name}/gw/gwd \
+                    -bd ${dbdir} \
+                    -hd ${langdir} \
+                    -lang en \
+                    -log ${logdir}/${name}.log \
+                    -daemon
+startupitem.pidfile none
+
+patchfiles          patch-bin-gwrepl-mk_data.ml.diff
+post-patch {
+    reinplace       "s|__MACPORTS_PREFIX__|${prefix}|g" \
+                    ${worksrcpath}/bin/gwrepl/mk_data.ml
 }
 
-pre-destroot		{
-	addgroup ${gwgroup}
-	set gid [existsgroup ${gwgroup}]
-	adduser ${gwuser} shell=/bin/sh gid=${gid} \
-		home=${prefix}/share/${name} \
-		realname=Geneweb\ Server
+configure {
+    system "cd ${worksrcpath} && ocaml ./configure.ml"
```

Use `system`'s `-W` argument rather than `cd`ing manually.
```suggestion
    system -W ${worksrcpath} "ocaml ./configure.ml"
```
Better yet, use the configure phase normally (specifying `configure.cmd`, `configure.args`, etc.) rather than overriding it.